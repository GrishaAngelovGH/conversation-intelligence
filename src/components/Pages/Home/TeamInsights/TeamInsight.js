import PropTypes from "prop-types"

const TeamInsight = ({ content, statistic }) => (
  <div className="row mt-3 mb-3 p-3 bg-light border rounded">
    <div className="col-md-12">

      <div className="row">
        <div className="col-md-12">
          {content}
        </div>
      </div>

      <div className="row mt-3 text-muted">
        <div className="col-md-12">
          {statistic}
        </div>
      </div>
    </div>
  </div>
)

TeamInsight.propTypes = {
  content: PropTypes.element.isRequired,
  statistic: PropTypes.element.isRequired
}

export default TeamInsight