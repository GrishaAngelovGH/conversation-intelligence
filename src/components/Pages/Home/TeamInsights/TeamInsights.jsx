import TeamInsight from './TeamInsight'

const TeamInsights = () => {
  const insights = [
    {
      id: 1,
      content: (<div><span className='text-info'>John Smith</span> and <span className='text-info'>Peter Potter</span> appear to talk the most, averaging <span className='fw-bold'>68%</span> during calls.</div>),
      statistic: (<div>This is <span className='fw-bold'>21%</span> higher than your top performers who average <span className='fw-bold'>56%</span> during calls.</div>)
    },
    {
      id: 2,
      content: (<div><span className='text-info'>Jane Smith</span> appears to talk the least, averaging <span className='fw-bold'>51%</span> during calls.</div>),
      statistic: (<div>This is <span className='fw-bold'>9%</span> lower than your top performers who average <span className='fw-bold'>56%</span> during calls.</div>)
    },
    {
      id: 3,
      content: (<div><span className='text-info'>Simon Doe</span> and <span className='text-info'>William Blake</span> have the lowest talking speed, averaging <span className='fw-bold'>118</span> words per minute.</div>),
      statistic: (<div>This is <span className='fw-bold'>21%</span> lower than your top performers who average <span className='fw-bold'>146</span> words per minute.</div>)
    }
  ]

  return (
    <div className='row mt-3'>
      <div className='col-md-12'>
        <h3 className='fw-bold'>Team Insights</h3>

        {
          insights.map(v => (
            <TeamInsight key={v.id} content={v.content} statistic={v.statistic} />
          ))
        }

      </div>
    </div>
  )
}

export default TeamInsights