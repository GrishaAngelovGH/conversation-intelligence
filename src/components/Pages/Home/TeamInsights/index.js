import TeamInsights from "./TeamInsights"

export default TeamInsights