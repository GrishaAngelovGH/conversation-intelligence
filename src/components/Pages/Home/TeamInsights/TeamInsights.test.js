import { render } from '@testing-library/react'

import TeamInsights from './TeamInsights'

test('should render TeamInsights component', () => {
  const view = render(<TeamInsights />)

  expect(view).toMatchSnapshot()
})