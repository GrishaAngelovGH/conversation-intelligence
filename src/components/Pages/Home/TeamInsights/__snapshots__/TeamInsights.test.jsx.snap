// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render TeamInsights component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row mt-3"
      >
        <div
          class="col-md-12"
        >
          <h3
            class="fw-bold"
          >
            Team Insights
          </h3>
          <div
            class="row mt-3 mb-3 p-3 bg-light border rounded"
          >
            <div
              class="col-md-12"
            >
              <div
                class="row"
              >
                <div
                  class="col-md-12"
                >
                  <div>
                    <span
                      class="text-info"
                    >
                      John Smith
                    </span>
                     and 
                    <span
                      class="text-info"
                    >
                      Peter Potter
                    </span>
                     appear to talk the most, averaging 
                    <span
                      class="fw-bold"
                    >
                      68%
                    </span>
                     during calls.
                  </div>
                </div>
              </div>
              <div
                class="row mt-3 text-muted"
              >
                <div
                  class="col-md-12"
                >
                  <div>
                    This is 
                    <span
                      class="fw-bold"
                    >
                      21%
                    </span>
                     higher than your top performers who average 
                    <span
                      class="fw-bold"
                    >
                      56%
                    </span>
                     during calls.
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="row mt-3 mb-3 p-3 bg-light border rounded"
          >
            <div
              class="col-md-12"
            >
              <div
                class="row"
              >
                <div
                  class="col-md-12"
                >
                  <div>
                    <span
                      class="text-info"
                    >
                      Jane Smith
                    </span>
                     appears to talk the least, averaging 
                    <span
                      class="fw-bold"
                    >
                      51%
                    </span>
                     during calls.
                  </div>
                </div>
              </div>
              <div
                class="row mt-3 text-muted"
              >
                <div
                  class="col-md-12"
                >
                  <div>
                    This is 
                    <span
                      class="fw-bold"
                    >
                      9%
                    </span>
                     lower than your top performers who average 
                    <span
                      class="fw-bold"
                    >
                      56%
                    </span>
                     during calls.
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="row mt-3 mb-3 p-3 bg-light border rounded"
          >
            <div
              class="col-md-12"
            >
              <div
                class="row"
              >
                <div
                  class="col-md-12"
                >
                  <div>
                    <span
                      class="text-info"
                    >
                      Simon Doe
                    </span>
                     and 
                    <span
                      class="text-info"
                    >
                      William Blake
                    </span>
                     have the lowest talking speed, averaging 
                    <span
                      class="fw-bold"
                    >
                      118
                    </span>
                     words per minute.
                  </div>
                </div>
              </div>
              <div
                class="row mt-3 text-muted"
              >
                <div
                  class="col-md-12"
                >
                  <div>
                    This is 
                    <span
                      class="fw-bold"
                    >
                      21%
                    </span>
                     lower than your top performers who average 
                    <span
                      class="fw-bold"
                    >
                      146
                    </span>
                     words per minute.
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row mt-3"
    >
      <div
        class="col-md-12"
      >
        <h3
          class="fw-bold"
        >
          Team Insights
        </h3>
        <div
          class="row mt-3 mb-3 p-3 bg-light border rounded"
        >
          <div
            class="col-md-12"
          >
            <div
              class="row"
            >
              <div
                class="col-md-12"
              >
                <div>
                  <span
                    class="text-info"
                  >
                    John Smith
                  </span>
                   and 
                  <span
                    class="text-info"
                  >
                    Peter Potter
                  </span>
                   appear to talk the most, averaging 
                  <span
                    class="fw-bold"
                  >
                    68%
                  </span>
                   during calls.
                </div>
              </div>
            </div>
            <div
              class="row mt-3 text-muted"
            >
              <div
                class="col-md-12"
              >
                <div>
                  This is 
                  <span
                    class="fw-bold"
                  >
                    21%
                  </span>
                   higher than your top performers who average 
                  <span
                    class="fw-bold"
                  >
                    56%
                  </span>
                   during calls.
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="row mt-3 mb-3 p-3 bg-light border rounded"
        >
          <div
            class="col-md-12"
          >
            <div
              class="row"
            >
              <div
                class="col-md-12"
              >
                <div>
                  <span
                    class="text-info"
                  >
                    Jane Smith
                  </span>
                   appears to talk the least, averaging 
                  <span
                    class="fw-bold"
                  >
                    51%
                  </span>
                   during calls.
                </div>
              </div>
            </div>
            <div
              class="row mt-3 text-muted"
            >
              <div
                class="col-md-12"
              >
                <div>
                  This is 
                  <span
                    class="fw-bold"
                  >
                    9%
                  </span>
                   lower than your top performers who average 
                  <span
                    class="fw-bold"
                  >
                    56%
                  </span>
                   during calls.
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="row mt-3 mb-3 p-3 bg-light border rounded"
        >
          <div
            class="col-md-12"
          >
            <div
              class="row"
            >
              <div
                class="col-md-12"
              >
                <div>
                  <span
                    class="text-info"
                  >
                    Simon Doe
                  </span>
                   and 
                  <span
                    class="text-info"
                  >
                    William Blake
                  </span>
                   have the lowest talking speed, averaging 
                  <span
                    class="fw-bold"
                  >
                    118
                  </span>
                   words per minute.
                </div>
              </div>
            </div>
            <div
              class="row mt-3 text-muted"
            >
              <div
                class="col-md-12"
              >
                <div>
                  This is 
                  <span
                    class="fw-bold"
                  >
                    21%
                  </span>
                   lower than your top performers who average 
                  <span
                    class="fw-bold"
                  >
                    146
                  </span>
                   words per minute.
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
